### DISCUSSION TITLE
Use phind search engine instead

### DISCUSSION THREAD

**Discussion by WhyIsEmerald** [dailydotdev/daily#1153] [Feature Requests]
Phind is a new search engine powered by AI and has latest info about tech so it would be nice if it came as default.

**Comment by nimrodkra** [comment]
Were happy to test it out but they don’t seem to have an API

**Reply by WhyIsEmerald** [reply]
It can be used as a search engine using URL so why not incorporate it using a URL

**Comment by nimrodkra** [comment]
Thanks for the suggestion, we'll consider it!

**Reply by WhyIsEmerald** [reply]
If you need any beta testers let me know  😉
